### ISSUE TITLE
Not pushing back fixes on private repo

### ISSUE THREAD

**Issue by marcelotrevisani-imgtec** [pre-commit/action#57]
Hi folks,

First of all, thank you very much for the ``pre-commit``! I really like to use it :)

The problem I am facing is that, I have a self-hosted machine for my private repository, it runs normally and it will accuse the problems, but it is not fixing the errors as it is mentioned on the docs. Maybe I am missing something, sorry if it is the case.

That is my yaml for pre-commit:
```yaml
name: pre-commit

on:
  pull_request:
  push:
    branches: [master]

jobs:
  pre-commit:
    runs-on: [self-hosted, linux, x64]
    steps:
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
    - uses: actions/setup-python@v2
    - uses: pre-commit/action@v2.0.0
      with:
        token: ${{ secrets.GITHUB_TOKEN }}
```

and my ``.pre-commit-config.yaml`` is:
```yaml
repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
    -   id: fix-encoding-pragma
        args: [--remove]
    -   id: sort-simple-yaml
    -   id: check-toml
    -   id: check-json
    -   id: check-merge-conflict
    -   id: pretty-format-json
        args: [--autofix]
    -   id: debug-statements
        language_version: python3
-   repo: https://github.com/pre-commit/mirrors-isort
    rev: v5.2.2
    hooks:
        -   id: isort
-   repo: https://gitlab.com/pycqa/flake8
    rev: 3.8.3
    hooks:
    -   id: flake8
        exclude: tests/data
        language_version: python3
        additional_dependencies:
            - flake8-typing-imports==1.9.0
            - flake8-builtins==1.5.3
            - flake8-bugbear==20.1.4
            - flake8-isort==3.0.1

```

Do you need any more information?

**Comment by marcelotrevisani-imgtec** [comment]
I forgot to add the full log of the serve:
```bash
2020-10-16T10:06:36.1820517Z ##[section]Starting: Request a runner to run this job
2020-10-16T10:06:36.3932504Z Found online and idle self-hosted runner in current repository that matches the required labels: 'self-hosted , nnatest , linux , x64'
2020-10-16T10:06:36.4322829Z ##[section]Finishing: Request a runner to run this job
2020-10-16T10:06:42.1215796Z Current runner version: '2.273.5'
2020-10-16T10:06:42.1218471Z Runner name: 'hhnnatest'
2020-10-16T10:06:42.1219352Z Machine name: 'hhnnatest'
2020-10-16T10:06:42.1221345Z Prepare workflow directory
2020-10-16T10:06:42.1367856Z Prepare all required actions
2020-10-16T10:06:42.1408303Z Getting action download info
2020-10-16T10:06:42.6243768Z Download action repository 'actions/checkout@v2'
2020-10-16T10:06:44.0954912Z Download action repository 'actions/setup-python@v2'
2020-10-16T10:06:44.6487945Z Download action repository 'pre-commit/action@v2.0.0'
2020-10-16T10:06:45.1898914Z ##[group]Run actions/checkout@v2
2020-10-16T10:06:45.1899323Z with:
2020-10-16T10:06:45.1899593Z   fetch-depth: 0
2020-10-16T10:06:45.1899874Z   repository: imaginationtech/nnatest
2020-10-16T10:06:45.1900511Z   token: ***
2020-10-16T10:06:45.1900704Z   ssh-strict: true
2020-10-16T10:06:45.1901021Z   persist-credentials: true
2020-10-16T10:06:45.1901305Z   clean: true
2020-10-16T10:06:45.1901551Z   lfs: false
2020-10-16T10:06:45.1901802Z   submodules: false
2020-10-16T10:06:45.1902042Z ##[endgroup]
2020-10-16T10:06:46.1065009Z Syncing repository: imaginationtech/nnatest
2020-10-16T10:06:46.1065565Z ##[group]Getting Git version info
2020-10-16T10:06:46.1066360Z Working directory is '/home/visionauto/actions-runner/_work/nnatest/nnatest'
2020-10-16T10:06:46.1066820Z [command]/usr/bin/git version
2020-10-16T10:06:46.1067083Z git version 2.25.1
2020-10-16T10:06:46.1067657Z ##[endgroup]
2020-10-16T10:06:46.1068145Z [command]/usr/bin/git config --local --get remote.origin.url
2020-10-16T10:06:46.1068649Z https://github.com/imaginationtech/nnatest
2020-10-16T10:06:46.1069167Z ##[group]Removing previously created refs, to avoid conflicts
2020-10-16T10:06:46.1069856Z [command]/usr/bin/git rev-parse --symbolic-full-name --verify --quiet HEAD
2020-10-16T10:06:46.1070243Z HEAD
2020-10-16T10:06:46.1070717Z [command]/usr/bin/git rev-parse --symbolic-full-name --branches
2020-10-16T10:06:46.1071087Z ##[endgroup]
2020-10-16T10:06:46.1071324Z ##[group]Cleaning the repository
2020-10-16T10:06:46.1071764Z [command]/usr/bin/git clean -ffdx
2020-10-16T10:06:46.3610538Z Removing .pytest_cache/
2020-10-16T10:06:46.3611501Z Removing 3/
2020-10-16T10:06:46.3612294Z Removing nnatest/__pycache__/
2020-10-16T10:06:46.3613278Z Removing nnatest/cli/__pycache__/
2020-10-16T10:06:46.3614338Z Removing nnatest/common/__pycache__/
2020-10-16T10:06:46.3615210Z Removing nnatest/flows/__pycache__/
2020-10-16T10:06:46.3616268Z Removing nnatest/flows/single_op_flows/__pycache__/
2020-10-16T10:06:46.3617390Z Removing nnatest/plugins/__pycache__/
2020-10-16T10:06:46.3618435Z Removing nnatest/plugins/metrics/__pycache__/
2020-10-16T10:06:46.3619649Z Removing nnatest/plugins/preprocessing/__pycache__/
2020-10-16T10:06:46.3620877Z Removing nnatest/plugins/single_op_plugins/__pycache__/
2020-10-16T10:06:46.3621940Z Removing nnatest/test/__pycache__/
2020-10-16T10:06:46.3623473Z Removing nnatest/test/sanity/__pycache__/
2020-10-16T10:06:46.3624566Z Removing nnatest/test/sanity/flows/__pycache__/
2020-10-16T10:06:46.3625570Z Removing nnatest/validation/__pycache__/
2020-10-16T10:06:46.3628370Z [command]/usr/bin/git reset --hard HEAD
2020-10-16T10:06:46.3673418Z HEAD is now at 1feac98 Merge edb9ad12640f1ea4ca4d0a53748eddb506318aae into c85d3e1b173ffe6789755a2a3087b2d4c8db264a
2020-10-16T10:06:46.3676549Z ##[endgroup]
2020-10-16T10:06:46.3680900Z ##[group]Disabling automatic garbage collection
2020-10-16T10:06:46.3682486Z [command]/usr/bin/git config --local gc.auto 0
2020-10-16T10:06:46.3702647Z ##[endgroup]
2020-10-16T10:06:46.3709001Z ##[group]Setting up auth
2020-10-16T10:06:46.3711943Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-10-16T10:06:46.3739703Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-10-16T10:06:46.3993009Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-10-16T10:06:46.4037514Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-10-16T10:06:46.4228381Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2020-10-16T10:06:46.4260482Z ##[endgroup]
2020-10-16T10:06:46.4262011Z ##[group]Fetching the repository
2020-10-16T10:06:46.4269198Z [command]/usr/bin/git -c protocol.version=2 fetch --prune --progress --no-recurse-submodules --unshallow origin +refs/heads/*:refs/remotes/origin/* +refs/tags/*:refs/tags/* +bf66ef820b207ea1fde6969fa98e07d1d585c4a5:refs/remotes/pull/7/merge
2020-10-16T10:06:47.7908615Z remote: Enumerating objects: 42, done.        
2020-10-16T10:06:47.7910106Z remote: Counting objects:   3% (1/33)        
2020-10-16T10:06:47.7910832Z remote: Counting objects:   6% (2/33)        
2020-10-16T10:06:47.7911562Z remote: Counting objects:   9% (3/33)        
2020-10-16T10:06:47.8013761Z remote: Counting objects:  12% (4/33)        
2020-10-16T10:06:47.8014755Z remote: Counting objects:  15% (5/33)        
2020-10-16T10:06:47.8015304Z remote: Counting objects:  18% (6/33)        
2020-10-16T10:06:47.8015578Z remote: Counting objects:  21% (7/33)        
2020-10-16T10:06:47.8015843Z remote: Counting objects:  24% (8/33)        
2020-10-16T10:06:47.8016111Z remote: Counting objects:  27% (9/33)        
2020-10-16T10:06:47.8017085Z remote: Counting objects:  30% (10/33)        
2020-10-16T10:06:47.8017537Z remote: Counting objects:  33% (11/33)        
2020-10-16T10:06:47.8017813Z remote: Counting objects:  36% (12/33)        
2020-10-16T10:06:47.8018083Z remote: Counting objects:  39% (13/33)        
2020-10-16T10:06:47.8018351Z remote: Counting objects:  42% (14/33)        
2020-10-16T10:06:47.8018617Z remote: Counting objects:  45% (15/33)        
2020-10-16T10:06:47.8018877Z remote: Counting objects:  48% (16/33)        
2020-10-16T10:06:47.8019158Z remote: Counting objects:  51% (17/33)        
2020-10-16T10:06:47.8019429Z remote: Counting objects:  54% (18/33)        
2020-10-16T10:06:47.8019698Z remote: Counting objects:  57% (19/33)        
2020-10-16T10:06:47.8019962Z remote: Counting objects:  60% (20/33)        
2020-10-16T10:06:47.8020225Z remote: Counting objects:  63% (21/33)        
2020-10-16T10:06:47.8020486Z remote: Counting objects:  66% (22/33)        
2020-10-16T10:06:47.8020750Z remote: Counting objects:  69% (23/33)        
2020-10-16T10:06:47.8021014Z remote: Counting objects:  72% (24/33)        
2020-10-16T10:06:47.8021280Z remote: Counting objects:  75% (25/33)        
2020-10-16T10:06:47.8021545Z remote: Counting objects:  78% (26/33)        
2020-10-16T10:06:47.8021810Z remote: Counting objects:  81% (27/33)        
2020-10-16T10:06:47.8022072Z remote: Counting objects:  84% (28/33)        
2020-10-16T10:06:47.8022335Z remote: Counting objects:  87% (29/33)        
2020-10-16T10:06:47.8022598Z remote: Counting objects:  90% (30/33)        
2020-10-16T10:06:47.8022869Z remote: Counting objects:  93% (31/33)        
2020-10-16T10:06:47.8023126Z remote: Counting objects:  96% (32/33)        
2020-10-16T10:06:47.8023391Z remote: Counting objects: 100% (33/33)        
2020-10-16T10:06:47.8023672Z remote: Counting objects: 100% (33/33), done.        
2020-10-16T10:06:47.8023975Z remote: Compressing objects:  10% (1/10)        
2020-10-16T10:06:47.8024275Z remote: Compressing objects:  20% (2/10)        
2020-10-16T10:06:47.8024568Z remote: Compressing objects:  30% (3/10)        
2020-10-16T10:06:47.8024858Z remote: Compressing objects:  40% (4/10)        
2020-10-16T10:06:47.8025147Z remote: Compressing objects:  50% (5/10)        
2020-10-16T10:06:47.8025443Z remote: Compressing objects:  60% (6/10)        
2020-10-16T10:06:47.8025726Z remote: Compressing objects:  70% (7/10)        
2020-10-16T10:06:47.8026021Z remote: Compressing objects:  80% (8/10)        
2020-10-16T10:06:47.8026314Z remote: Compressing objects:  90% (9/10)        
2020-10-16T10:06:47.8026865Z remote: Compressing objects: 100% (10/10)        
2020-10-16T10:06:47.8027173Z remote: Compressing objects: 100% (10/10), done.        
2020-10-16T10:06:47.8602581Z remote: Total 17 (delta 7), reused 13 (delta 5), pack-reused 0        
2020-10-16T10:06:47.8990792Z From https://github.com/imaginationtech/nnatest
2020-10-16T10:06:47.8991898Z  + 1feac98...bf66ef8 bf66ef820b207ea1fde6969fa98e07d1d585c4a5 -> pull/7/merge  (forced update)
2020-10-16T10:06:47.9039597Z ##[endgroup]
2020-10-16T10:06:47.9041433Z ##[group]Determining the checkout info
2020-10-16T10:06:47.9042680Z ##[endgroup]
2020-10-16T10:06:47.9043686Z ##[group]Checking out the ref
2020-10-16T10:06:47.9046030Z [command]/usr/bin/git checkout --progress --force refs/remotes/pull/7/merge
2020-10-16T10:06:47.9075736Z Warning: you are leaving 1 commit behind, not connected to
2020-10-16T10:06:47.9077152Z any of your branches:
2020-10-16T10:06:47.9077907Z 
2020-10-16T10:06:47.9079770Z   1feac98 Merge edb9ad12640f1ea4ca4d0a53748eddb506318aae into c85d3e1b173ffe6789755a2a3087b2d4c8db264a
2020-10-16T10:06:47.9081566Z 
2020-10-16T10:06:47.9082676Z If you want to keep it by creating a new branch, this may be a good time
2020-10-16T10:06:47.9083847Z to do so with:
2020-10-16T10:06:47.9084510Z 
2020-10-16T10:06:47.9086129Z  git branch <new-branch-name> 1feac98
2020-10-16T10:06:47.9087064Z 
2020-10-16T10:06:47.9089222Z HEAD is now at bf66ef8 Merge 223b093c326fe06e92f54828f2525cff3c99f7fd into c85d3e1b173ffe6789755a2a3087b2d4c8db264a
2020-10-16T10:06:47.9092383Z ##[endgroup]
2020-10-16T10:06:47.9148126Z [command]/usr/bin/git log -1 --format='%H'
2020-10-16T10:06:47.9168220Z 'bf66ef820b207ea1fde6969fa98e07d1d585c4a5'
2020-10-16T10:06:47.9268249Z ##[group]Run actions/setup-python@v2
2020-10-16T10:06:47.9268576Z with:
2020-10-16T10:06:47.9268841Z   python-version: 3.x
2020-10-16T10:06:47.9269433Z   token: ***
2020-10-16T10:06:47.9269666Z ##[endgroup]
2020-10-16T10:06:47.9699415Z Successfully setup CPython (3.8.6)
2020-10-16T10:06:47.9827188Z ##[group]Run pre-commit/action@v2.0.0
2020-10-16T10:06:47.9827908Z with:
2020-10-16T10:06:47.9829241Z   token: ***
2020-10-16T10:06:47.9829844Z   extra_args: --all-files
2020-10-16T10:06:47.9830414Z env:
2020-10-16T10:06:47.9831398Z   pythonLocation: /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64
2020-10-16T10:06:47.9832872Z   LD_LIBRARY_PATH: /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib
2020-10-16T10:06:47.9833886Z ##[endgroup]
2020-10-16T10:06:48.0261598Z ##[group]install pre-commit
2020-10-16T10:06:48.0305498Z [command]/home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/bin/pip install pre-commit
2020-10-16T10:06:48.3245068Z Requirement already satisfied: pre-commit in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (2.7.1)
2020-10-16T10:06:48.3332996Z Requirement already satisfied: virtualenv>=20.0.8 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from pre-commit) (20.0.35)
2020-10-16T10:06:48.3593209Z Requirement already satisfied: nodeenv>=0.11.1 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from pre-commit) (1.5.0)
2020-10-16T10:06:48.3689787Z Requirement already satisfied: identify>=1.0.0 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from pre-commit) (1.5.6)
2020-10-16T10:06:48.3691135Z Requirement already satisfied: pyyaml>=5.1 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from pre-commit) (5.3.1)
2020-10-16T10:06:48.3692394Z Requirement already satisfied: toml in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from pre-commit) (0.10.1)
2020-10-16T10:06:48.3693562Z Requirement already satisfied: cfgv>=2.0.0 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from pre-commit) (3.2.0)
2020-10-16T10:06:48.3694886Z Requirement already satisfied: appdirs<2,>=1.4.3 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from virtualenv>=20.0.8->pre-commit) (1.4.4)
2020-10-16T10:06:48.3696449Z Requirement already satisfied: six<2,>=1.9.0 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from virtualenv>=20.0.8->pre-commit) (1.15.0)
2020-10-16T10:06:48.3697641Z Requirement already satisfied: distlib<1,>=0.3.1 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from virtualenv>=20.0.8->pre-commit) (0.3.1)
2020-10-16T10:06:48.3698840Z Requirement already satisfied: filelock<4,>=3.0.0 in /home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/lib/python3.8/site-packages (from virtualenv>=20.0.8->pre-commit) (3.0.12)
2020-10-16T10:06:48.4193290Z [command]/home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/bin/pip freeze --local
2020-10-16T10:06:48.6086277Z appdirs==1.4.4
2020-10-16T10:06:48.6086997Z cfgv==3.2.0
2020-10-16T10:06:48.6087608Z distlib==0.3.1
2020-10-16T10:06:48.6089105Z filelock==3.0.12
2020-10-16T10:06:48.6089680Z identify==1.5.6
2020-10-16T10:06:48.6090232Z nodeenv==1.5.0
2020-10-16T10:06:48.6091620Z pre-commit==2.7.1
2020-10-16T10:06:48.6092205Z PyYAML==5.3.1
2020-10-16T10:06:48.6092705Z six==1.15.0
2020-10-16T10:06:48.6093176Z toml==0.10.1
2020-10-16T10:06:48.6093725Z virtualenv==20.0.35
2020-10-16T10:06:48.6329739Z ##[endgroup]
2020-10-16T10:07:02.8194988Z Cache Size: ~16 MB (17266073 B)
2020-10-16T10:07:02.8242489Z [command]/usr/bin/tar -z -xf /home/visionauto/actions-runner/_work/_temp/cd7c7dc4-b696-4c51-b2ce-2e3eebcf378b/cache.tgz -P -C /home/visionauto/actions-runner/_work/nnatest/nnatest
2020-10-16T10:07:03.2214526Z [command]/home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/bin/pre-commit run --show-diff-on-failure --color=always --all-files
2020-10-16T10:07:03.4834149Z Trim Trailing Whitespace.................................................[42mPassed[m
2020-10-16T10:07:03.5325734Z Fix End of Files.........................................................[41mFailed[m
2020-10-16T10:07:03.5338808Z [2m- hook id: end-of-file-fixer[m
2020-10-16T10:07:03.5339847Z [2m- exit code: 1[m
2020-10-16T10:07:03.5340465Z [2m- files were modified by this hook[m
2020-10-16T10:07:03.5340869Z 
2020-10-16T10:07:03.5341309Z Fixing nnatest/cli/output.py
2020-10-16T10:07:03.5341648Z 
2020-10-16T10:07:03.5812555Z Fix python encoding pragma...............................................[42mPassed[m
2020-10-16T10:07:03.5813303Z Sort simple YAML files...............................(no files to check)[46;30mSkipped[m
2020-10-16T10:07:03.5813911Z Check Toml...........................................(no files to check)[46;30mSkipped[m
2020-10-16T10:07:03.6228831Z Check JSON...............................................................[42mPassed[m
2020-10-16T10:07:03.6632201Z Check for merge conflicts................................................[42mPassed[m
2020-10-16T10:07:03.7049757Z Pretty format JSON.......................................................[42mPassed[m
2020-10-16T10:07:04.5972860Z Debug Statements (Python)................................................[42mPassed[m
2020-10-16T10:07:04.5973913Z isort....................................................................[41mFailed[m
2020-10-16T10:07:04.6049771Z [2m- hook id: isort[m
2020-10-16T10:07:04.6050207Z [2m- files were modified by this hook[m
2020-10-16T10:07:04.6050415Z 
2020-10-16T10:07:04.6050919Z Fixing /home/visionauto/actions-runner/_work/nnatest/nnatest/nnatest/cli/output.py
2020-10-16T10:07:04.6051229Z 
2020-10-16T10:07:11.4441648Z flake8...................................................................[42mPassed[m
2020-10-16T10:07:11.4442639Z pre-commit hook(s) made changes.
2020-10-16T10:07:11.4443454Z If you are seeing this message in CI, reproduce locally with: `pre-commit run --all-files`.
2020-10-16T10:07:11.4444209Z To run `pre-commit` as part of git workflow, use `pre-commit install`.
2020-10-16T10:07:11.4445361Z All changes made by hooks:
2020-10-16T10:07:11.4530626Z [1mdiff --git a/nnatest/cli/output.py b/nnatest/cli/output.py[m
2020-10-16T10:07:11.4531189Z [1mindex fe2bd2f..cf45d3f 100644[m
2020-10-16T10:07:11.4531580Z [1m--- a/nnatest/cli/output.py[m
2020-10-16T10:07:11.4531957Z [1m+++ b/nnatest/cli/output.py[m
2020-10-16T10:07:11.4532275Z [36m@@ -1,4 +1,5 @@[m
2020-10-16T10:07:11.4532621Z  from colorama import Fore, Style[m
2020-10-16T10:07:11.4533010Z [32m+[m
2020-10-16T10:07:11.4533300Z  import nnatest[m
2020-10-16T10:07:11.4533616Z  [m
2020-10-16T10:07:11.4533863Z  [m
2020-10-16T10:07:11.4534224Z [36m@@ -27,5 +28,3 @@[m [mdef print_killed_jobs(list_jobs):[m
2020-10-16T10:07:11.4534701Z      print(f"{Style.BRIGHT}{Fore.CYAN}-- Jobs killed on TFC --")[m
2020-10-16T10:07:11.4535117Z      for job in list_jobs:[m
2020-10-16T10:07:11.4535635Z          print(f"{Style.NORMAL}{Fore.LIGHTBLACK_EX}  - {Fore.LIGHTCYAN_EX}{job}")[m
2020-10-16T10:07:11.4536103Z [31m-[m
2020-10-16T10:07:11.4536399Z [31m-[m
2020-10-16T10:07:11.4676157Z [command]/home/visionauto/actions-runner/_work/_tool/Python/3.8.6/x64/bin/pre-commit run --show-diff-on-failure --color=always --all-files
2020-10-16T10:07:11.7485452Z Trim Trailing Whitespace.................................................[42mPassed[m
2020-10-16T10:07:11.7948012Z Fix End of Files.........................................................[42mPassed[m
2020-10-16T10:07:11.8423417Z Fix python encoding pragma...............................................[42mPassed[m
2020-10-16T10:07:11.8430768Z Sort simple YAML files...............................(no files to check)[46;30mSkipped[m
2020-10-16T10:07:11.8439641Z Check Toml...........................................(no files to check)[46;30mSkipped[m
2020-10-16T10:07:11.8878436Z Check JSON...............................................................[42mPassed[m
2020-10-16T10:07:11.9293568Z Check for merge conflicts................................................[42mPassed[m
2020-10-16T10:07:12.0052421Z Pretty format JSON.......................................................[42mPassed[m
2020-10-16T10:07:12.3062292Z Debug Statements (Python)................................................[42mPassed[m
2020-10-16T10:07:12.7835719Z isort....................................................................[42mPassed[m
2020-10-16T10:07:19.7333597Z flake8...................................................................[42mPassed[m
2020-10-16T10:07:19.7550354Z [command]/usr/bin/git diff --quiet
2020-10-16T10:07:19.7576962Z ##[group]push fixes
2020-10-16T10:07:19.7586186Z [command]/usr/bin/git config user.name pre-commit
2020-10-16T10:07:19.7631807Z [command]/usr/bin/git config user.email pre-commit@example.com
2020-10-16T10:07:19.7662802Z [command]/usr/bin/git checkout HEAD -b fix-gha
2020-10-16T10:07:19.7718649Z Switched to a new branch 'fix-gha'
2020-10-16T10:07:19.7721169Z M	nnatest/cli/output.py
2020-10-16T10:07:19.7730086Z [command]/usr/bin/git commit -am pre-commit fixes
2020-10-16T10:07:19.7813817Z [fix-gha e9fb376] pre-commit fixes
2020-10-16T10:07:19.7815384Z  1 file changed, 1 insertion(+), 2 deletions(-)
2020-10-16T10:07:19.7840222Z [command]/usr/bin/git remote set-url origin ***github.com/marcelotrevisani-imgtec/nnatest.git
2020-10-16T10:07:19.7871523Z [command]/usr/bin/git push origin HEAD
2020-10-16T10:07:20.2742472Z remote: Repository not found.
2020-10-16T10:07:20.2800763Z fatal: repository 'https://github.com/marcelotrevisani-imgtec/nnatest.git/' not found
2020-10-16T10:07:20.2801873Z ##[endgroup]
2020-10-16T10:07:20.2804660Z ##[error]The process '/usr/bin/git' failed with exit code 128
2020-10-16T10:07:20.2876255Z Post job cleanup.
2020-10-16T10:07:20.3717992Z [command]/usr/bin/git version
2020-10-16T10:07:20.3768360Z git version 2.25.1
2020-10-16T10:07:20.3782529Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-10-16T10:07:20.3810006Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-10-16T10:07:20.4007188Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-10-16T10:07:20.4035912Z http.https://github.com/.extraheader
2020-10-16T10:07:20.4049491Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-10-16T10:07:20.4081370Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-10-16T10:07:20.4286980Z Cleaning up orphan processes

```

It seems it is not finding the repository... maybe it is because I opened a PR from a fork?

**Comment by asottile** [comment]
> maybe it is because I opened a PR from a fork?

correct -- GA does not expose sensitive information to forks
